endpoints/openrtb2/auction.go

```diff
@@ -1661,11 +1661,7 @@ func (deps *endpointDeps) processStoredRequests(ctx context.Context, requestJson
 	resolvedRequest := requestJson
 
 	if hasStoredBidRequest {
-		isAppRequest, err := checkIfAppRequest(requestJson)
-		if err != nil {
-			return nil, nil, []error{err}
-		}
-		if isAppRequest && (deps.cfg.GenerateRequestID || bidRequestID == "{{UUID}}") {
+		if deps.cfg.GenerateRequestID || bidRequestID == "{{UUID}}" {
```

Comment:
The original feature request changed a few times, but I believe we decided to generate a new bid request id for app and amp auctions. I'm good with expanding this to any request type if the special {{UUID}} token is used in the stored request, but the deps.cfg.GenerateRequestID should still be scoped to app and amp requests, IMHO.


deps.cfg.GenerateRequestID || bidRequestID == "{{UUID}}"